tools/mdm/apple/demo.md

```diff
@@ -179,24 +172,24 @@ fleetctl apple-mdm enrollment-profiles create-automatic --dep-profile ./tools/md
 Automatic enrollment profile created, ID: 2
 ```
 
-## 9. DEP Enroll
+## 8. DEP Enroll
 
-1. Assign the device to our MDM server in https://business.apple.com
-2. Fleet should pick it up and assign an DEP enroll profile that points to itself (must wait for 5m after executing the `create-automatic` command).
-3. Start the VM and enroll.
+Follow https://travellingtechguy.blog/vmware-dep/ to create a VMWare macOS VM with a serial number that Fleet's ABM manages. (Reach out to Guillaume, Michal or Zach Wasserman.)
 
-### Examples
+1. Assign the device to our "MDM server" in https://business.apple.com
+2. Fleet should pick it up and assign an DEP enroll profile that points to itself (must wait for one minute after executing the `create-automatic` command).
+3. Start the VM.
 
-Install profile on a device:
+### Examples
 
+Install a profile to allow for enabling debug mode of MDM on a device (useful when developing/testing MDM features):
```

Comment:
We should probably include an example of how to get the device id for the following command.

Reply:
Good idea. Done.